src/content/plugins/split-chunks-plugin.md

```diff
@@ -199,12 +199,27 @@ module.exports = {
 
 A module can belong to multiple cache groups. The optimization will prefer the cache group with a higher `priority`. The default groups have a negative priority to allow custom groups to take higher priority (default value is `0` for custom groups).
 
-#### `splitChunks.cacheGroups.reuseExistingChunk`
+#### `splitChunks.cacheGroups.cacheGroup.reuseExistingChunk`
```

Comment:
No `{cacheGroup}` notation?

Reply:
i am tempted to add it, but i think there was another similar PR about `test` option and we did cacheGroup there, or is it still open?